### PR TITLE
pkg/volume/portworx: Update vendor for libopenstorage/openstorage to v1.0.0

### LINKED ISSUE #80029: Update libopenstorage/openstorage dependency to a release version
Found this when trying to see which vendored dependencies does not use a proper tag (and is using a random SHA). 

The https://github.com/libopenstorage/openstorage dependency is very old and there have been a lot of updates to it:
Looks like the https://github.com/libopenstorage/openstorage/compare/093a0c388875...v8.0.0

However there does not seem to be anyone around to take care of the portworx plugin. Is it time to deprecate it? is anyone using it?

/sig storage

### PR DESCRIPTION
**Pull request by adityadani**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
vendor: update for libopenstorage/openstorage to v1.0.0  
- Portworx storage plugin under pkg/volume/portworx uses the libopenstorage/openstorage dependency.
- This change does not update any of the go files vendor'ed from libopenstorage/openstorage but simply updates the go modules vendor from a SHA to the v1.0.0 tag.

This PR addresses action item 2 and 3, and partially address action item 1 from this comment:
https://github.com/kubernetes/kubernetes/pull/80281#issuecomment-513986145

**Which issue(s) this PR fixes**:
Fixes #80029 

**Special notes for your reviewer**:
The changes are divided into two commits:
1. Updates the dependencies to use tag v1.0.0 instead of a sha to libopenstorage/openstorage
2. Update the OWNERS for pkg/volume/portworx

**Does this PR introduce a user-facing change?**:

```release-note
Update portworx plugin dependency on libopenstorage/openstorage to v1.0.0
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

/sig storage
/area code-organization

### CODE REVIEW

**Comment by adityadani** [comment]
cc @lpabon @saad-ali

File: pkg/volume/portworx/OWNERS [hunk 1]
```diff
@@ -2,3 +2,4 @@
 
 maintainers:
 - adityadani
+- lpabon
```
Comment (neolit123) [inline pkg/volume/portworx/OWNERS]: the OWNERS files use the `approvers/reviewers` lists.

see this file for example:
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/flocker/OWNERS

`maintainers` is AFAIK not recognized.

**Comment by saad-ali** [comment]
/assign

Discussion k8s membership offline with @adityadani and @lpabon

**Comment by dims** [comment]
/approve
/lgtm

@adityadani once @lpabon accepts the org membership, please rebase this to get rid of `do-not-merge/invalid-owners-file`

**Comment by dims** [comment]
/assign @thockin @liggitt 
/approve
/lgtm

Tim, Jordan, easy one, no change in code, the random SHA was tagged as `1.0.0`, needs root OWNERS approval

**Pull request merged**